release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -0,0 +1,1151 @@
+:_content-type: ASSEMBLY
+[id="ocp-4-13-release-notes"]
+= {product-title} {product-version} release notes
+include::_attributes/common-attributes.adoc[]
+:context: release-notes
+
+toc::[]
+
+Red Hat {product-title} provides developers and IT organizations with a hybrid cloud application platform for deploying both new and existing applications on secure, scalable resources with minimal configuration and management overhead. {product-title} supports a wide selection of programming languages and frameworks, such as Java, JavaScript, Python, Ruby, and PHP.
+
+Built on {op-system-base-full} and Kubernetes, {product-title} provides a more secure and scalable multitenant operating system for today's enterprise-class applications, while delivering integrated application runtimes and libraries. {product-title} enables organizations to meet security, privacy, compliance, and governance requirements.
+
+[id="ocp-4-13-about-this-release"]
+== About this release
+
+// TODO: Update with the relevant information closer to release.
+{product-title} (link:https://access.redhat.com/errata/RHSA-2022:7399[RHSA-2022:7399]) is now available. This release uses link:https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.25.md[Kubernetes 1.25] with CRI-O runtime. New features, changes, and known issues that pertain to {product-title} {product-version} are included in this topic.
+
+{product-title} {product-version} clusters are available at https://console.redhat.com/openshift. With the {cluster-manager-first} application for {product-title}, you can deploy OpenShift clusters to either on-premises or cloud environments.
+
+// Double check OP system versions
+{product-title} {product-version} is supported on {op-system-base-full} 8.4 and 8.5, as well as on {op-system-first} 4.13.
+
+You must use {op-system} machines for the control plane, and you can use either {op-system} or {op-system-base} for compute machines.
+//Removed the note per https://issues.redhat.com/browse/GRPA-3517
+
+//TODO: Remove this for 4.13
```

Comment:
Don't forget to address this whenever appropriate.

Reply:
Will do